SEO improvements to guides (#2915)
gradio-app/gradio: guides/01_getting-started/02_key-features.md

# Key Features

> Let's go through some of the most popular features of Gradio! 
> 
> Here are Gradio's key features: 

Reviewer: Can we shorten this? Suggestion:
```suggestion
Let's go through some key features you can add to your Gradio demo:
```
